setenv.bash

```diff
@@ -9,7 +9,7 @@ DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 if [[ "$OSTYPE" == "cygwin" ]]; 
 then
-  go env | grep GOARCH | cut -d = -f 2 | grep 386 || die "Lantern on Windows requires Go for 386. Please reinstall from an installer at https://golang.org/dl/ or build from source"
+  export GOPATH=386 # Requires go1.5+
```

Comment:
We can use a different arch since go1.5, and go1.6 is already a requisite on the README.